File: ios/Poste_iOS/Poste_iOS/Network/DTO/PostDTO.swift
import Foundation


struct PostDTO: Codable {
    
    let id: String
    let folderId: String?
    let title: String
    let url: String?
    let notes: String?
    let tags: [String]?
    let createdAt: String?
    let deletedAt: String?
    
}

File: multiplatform/multiPlatform/composeApp/src/commonMain/kotlin/network/dto/PostDto.kt
package network.dto

import kotlinx.serialization.Serializable

/**
 * DTO for Post responses
 */
@Serializable
data class PostDto(
    val id: String,
    val folderId: String,
    val title: String,
    val url: String,
    val notes: String? = null,
    val tags: List<String> = emptyList(),
    val createdAt: String,
    val deletedAt: String? = null
)

